src/Polly/RateLimit/AsyncRateLimitSyntax.cs

```diff
@@ -10,12 +10,14 @@ public partial class Policy
         /// </summary>
         /// <param name="numberOfExecutions">The number of executions (call it N) permitted per timespan.</param>
         /// <param name="perTimeSpan">How often N executions are permitted.</param>
+        /// <param name="spreadUniformly">The number of executions allowed are spread equally within the specified time span, default is true.</param>
         /// <returns>The policy instance.</returns>
         public static AsyncRateLimitPolicy RateLimitAsync(
             int numberOfExecutions,
-            TimeSpan perTimeSpan)
+            TimeSpan perTimeSpan,
+            bool spreadUniformly = true)
```

Comment:
Adding these optional parameters is a binary breaking change, so additional overloads would be preferred to remove the need to bump to v8.